TheThingsNetwork/lorawan-stack
# Simplify Packet Broker settings

#### Summary



Closes [#3838](https://github.com/TheThingsIndustries/lorawan-stack/issues/3838)

#### Changes



- Update packet broker main view
- Separate two forms
- Add radio button options
- Update e2e tests
- Remove gateway visibility

![Screenshot 2023-10-18 at 14-47-15 Management platform for The Things Stack for LoRaWAN](https://github.com/TheThingsNetwork/lorawan-stack/assets/56658938/5c4a0d5a-1f6d-462e-83ae-81f2d7dd6e59)

#### Testing



1. Open packet broker in the admin panel.
2. Select the first two options one at a time to make sure each one is successful.
3. Select custom configuration, go to the network tab and change a network config.
4. Reload the page see that the configuration you selected is the one shown.


#### Checklist
